Editor/AGS.Editor/Components/StatisticsSenderComponent.cs

```diff
@@ -15,16 +15,14 @@ internal class StatisticsSenderComponent : BaseComponent
         private readonly int _screenWidth;
         private readonly int _screenHeight;
 
-        private Timer _timer;
-
         public StatisticsSenderComponent(GUIController guiController, AGSEditor agsEditor)
             : base(guiController, agsEditor)
         {
             // Screen width x height must be cached here, otherwise we could attempt to
             // retrieve them while the game is being tested full-screen
             _screenWidth = System.Windows.Forms.SystemInformation.PrimaryMonitorSize.Width;
             _screenHeight = System.Windows.Forms.SystemInformation.PrimaryMonitorSize.Height;
-            _timer = new Timer(Timer_Callback, null, 120000, 240000);
+            new Timer(Timer_Callback, null, 120000, 240000);

```

Comment:
That's a really good question, I don't know enough of the nitty gritty details of the automatic garbage collector, in my mind it makes sense that it wouldn't dispose a value in memory that is still being used. Just to be safe I suppose we could disable the error throw here with a pragma statement instead.